client/app/competitor/page.tsx

```diff
@@ -115,41 +120,19 @@ const TestResults = () => {
                         <div className="flex h-full flex-grow flex-col gap-2">
                             <b>Input</b>
                             <pre className="w-full rounded-sm bg-slate-800 px-4 py-2 font-mono text-white">
-                                2 11 15 0<br />
-                                2 11 15 0<br />
-                                2 11 15 0<br />
-                                2 11 15 0<br />
-                                2 11 15 0<br />2 11 15 0
+                                {input}
                             </pre>
                         </div>
                         <div className="flex h-full flex-grow flex-col gap-2">
-                            <b>Expected Output</b>
+                            <b>Output</b>
                             <pre className="w-full rounded-sm bg-slate-800 px-4 py-2 font-mono text-white">
-                                0 2 11 15
-                                <br />
-                                0 2 11 15
-                                <br />
-                                0 2 11 15
-                                <br />
-                                0 2 11 15
-                                <br />
-                                0 2 11 15
-                                <br />0 2 11 15
+                                {output}
                             </pre>
                         </div>
                         <div className="flex h-full flex-grow flex-col gap-2">
                             <b>Expected Output</b>
                             <pre className="w-full rounded-sm bg-slate-800 px-4 py-2 font-mono text-white">
-                                0 2 11 15
-                                <br />
-                                0 2 11 15
-                                <br />
-                                0 2 11 15
-                                <br />
-                                0 2 11 15
-                                <br />
-                                0 2 11 15
-                                <br />0 2 11 15
+                                {expectedOutput}
                             </pre>
                         </div>
                     </AccordionContent>
```

Comment:
Have this read from an array to get the input/output/expected output as that's what we'll need to be when we get test output.  That will also mean we don't need to repeat this and have our current styling issues.

Reply:
This needs to read from an array _instead_ of listing out each test case.  We need to be able to support a number of test cases and show the content dynamically.